database/seeds/UpgradeH5pVersionsSeeder.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class UpgradeH5pVersionsSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $currentDate = now();
+
+        $this->branchingScenarioSetDemoIdAndUrl($currentDate);
+
+
+
+        // update course presentation 1.22 to 1.24 in activity layouts
+        $coursePresentationActivityLayoutsParams = ['title' => 'Course Presentation', 'h5pLib' => 'H5P.CoursePresentation 1.22'];
+        $coursePresentationActivityLayouts = DB::table('activity_layouts')->where($coursePresentationActivityLayoutsParams)->first();
+        
+        if ($coursePresentationActivityLayouts) {
+            DB::table('activity_layouts')->where($coursePresentationActivityLayoutsParams)->update([
+                'h5pLib' => 'H5P.CoursePresentation 1.24',
+                'updated_at' => $currentDate
+            ]);
+        }
+
+
+        // update fill in the blanks 1.12 to 1.14 in activity items
+        $fillInTheBlanksActivityItemsParams = ['title' => 'Fill In The Blanks', 'h5pLib' => 'H5P.Blanks 1.12'];
+        $fillInTheBlanksActivityItems = DB::table('activity_items')->where($fillInTheBlanksActivityItemsParams)->first();
+        if ($fillInTheBlanksActivityItems) {
+            DB::table('activity_items')->where($fillInTheBlanksActivityItemsParams)->update([
+                'h5pLib' => 'H5P.Blanks 1.14',
+                'updated_at' => $currentDate
+            ]);
+        }
+
+        // update true false 1.6 to 1.8 in activity items
+        $trueFalseActivityItemsParams = ['title' => 'True & False', 'h5pLib' => 'H5P.TrueFalse 1.6'];
```

Comment:
you can do the update operation for single table(activity_items) like below to avoid to RUN select and update query multiple times:

$activity_items = [
            'Fill In The Blanks' => 'H5P.Blanks 1.14',
            'True & False' => 'H5P.TrueFalse 1.8',
            'Drag & Drop' => 'H5P.DragQuestion 1.14',
            // so on
        ];

        foreach ($activity_items as $title => $h5pLib) {
            DB::table('activity_items')
                ->where('title', $title)
                ->update([
                    'h5pLib' => $h5pLib,
                    'updated_at' => $currentDate
                ]);
        }